.gitignore

```diff
@@ -6,3 +6,4 @@ lib
 node_modules
 npm-debug.log
 coverage
+.idea
```

Comment:
Yeah. I know about this feature. But lot of people use IntelliJ products and it can be convenient. React has Sublime in ignore list, for example https://github.com/enaqx/react/blob/master/.gitignore#L24